.pre-commit-config.yaml

```diff
@@ -39,7 +39,10 @@ repos:
     rev: "v3.5.3"
     hooks:
       - id: prettier
-        types_or: [yaml, markdown, html, css, scss, javascript, json]
+        types_or: [yaml, html, css, scss, javascript, json]
+      - id: prettier
+        types_or: [markdown]
+        args: ["--prose-wrap=always"]
```

Comment:
the problem is that we really need to be careful with the spacing in `:::` blocks... this is why i removed it at some point. we can try to be careful

Reply:
we could also add a pygrep hook to catch `:::` at the end of lines, but not not on a line itself?

the start tags get ugly when not on their own line, but should typically still work. Only the end tags work only on their own line.

Reply:
yeah maybe, but i'm fine with merging this for now